src/components/NgoDetails/index.jsx

```diff
@@ -2,19 +2,28 @@ import React from "react";
 import { Button } from "react-bootstrap";
 import "./index.css";
 import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
 
 const NgoDetails = ({ details }) => {
   const { t } = useTranslation();
-
+  const initialLanguage = useSelector((state) => state.languageReducer);
+  const renderDetails = () => {
+    console.log(initialLanguage);
+    switch (initialLanguage) {
+      case "ar":
+        return details.serviceDescription.arabic;
```

Comment:
these won't be needed I think cause the i18n automaticaly transaltes it. all u have to do is have details.serviceDescription in the 3 json files and it changes it auto

Reply:
by saying 3 json files i mean each one for the ar eng or tr

Reply:
are the ngo details coming for the json files in i18n or is it fetched with firebase ?

Reply:
oh okai its coming from firebase, ignore what i just said.

Reply:
It comes from the database not from the translation